berth_reservations/settings.py

```diff
@@ -132,6 +132,8 @@
 
 LOCALE_PATHS = (os.path.join(BASE_DIR, "locale"),)
 
+REQUEST_IGNORE_PATHS = ("/healthz", "/readiness")
```

Comment:
Could rename to `REQUEST_LOGGER_IGNORE_PATHS` or similar. Might otherwise be confusing at first glance.

Reply:
Good one, I didn't have any better ideas for names 😅